Extension/src/LanguageServer/extension.ts

```diff
@@ -796,9 +796,24 @@ export function registerCommands(): void {
     disposables.push(vscode.commands.registerCommand('cpptools.activeConfigName', onGetActiveConfigName));
     disposables.push(vscode.commands.registerCommand('cpptools.activeConfigCustomVariable', onGetActiveConfigCustomVariable));
     disposables.push(vscode.commands.registerCommand('cpptools.setActiveConfigName', onSetActiveConfigName));
+    disposables.push(vscode.commands.registerCommand('C_Cpp.RestartLanguageServerActiveFile', onRestartLanguageServerActiveFile));
+
     getTemporaryCommandRegistrarInstance().executeDelayedCommands();
 }
 
+async function onRestartLanguageServerActiveFile(): Promise<void> {
+    const activeEditor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+    if (!activeEditor || !activeEditor.document) {
```

Comment:
should `activeEditor.document.uri.scheme !== "file"` also be added to this conditional?

see function `onDidChangeActiveTextEditor` for example.